app/uk/gov/hmrc/contactadvisors/controllers/SecureMessageController.scala

```diff
@@ -47,6 +47,15 @@ class SecureMessageController @Inject()(customerAdviceAudit: CustomerAdviceAudit
     )
   }
 
+  def inbox_v2 = Action.async { implicit request =>
```

Comment:
I'd suggest maybe sticking to camel case as opposed to snake case for method names as snake casing is usually used for views/template names.

Reply:
Done thanks.